imap/imapd.c

```diff
@@ -1598,6 +1609,7 @@ static void cmdloop(void)
         /* Parse command name */
         c = getword(imapd_in, &cmd);
         if (!cmd.s[0]) {
+            syntax_errors ++;
```

Comment:
I go back and forth... if it were any other equation there'd be a space before the operator, so smashing ++ on there without one looks odd.  On the other hand, `i++` specifically is very idiomatic, and so `foo ++` looks odd in comparison, especially if the two are nearby.  I think I somewhat prefer the space for longer variable names.  I usually mimic what's nearby if there's an example nearby, and use what feels right otherwise.